tests/language/sync_star/generator2_test.dart

```diff
@@ -8,7 +8,7 @@ import "package:expect/expect.dart";
 import "dart:async";
 
 var sync = "topLevelSync";
-var async = "topLevelAync";
+var async = "topLevelASync";
```

Comment:
Notable. I'm fairly certain this is correct.

Reply:
Should the `S` be lower-case?

Reply:
Fair. I'm not certain about the casing.

Reply:
Actually, yes, my natural inclination would be that it should be lowercase.

Reply:
Yes, lowercase `S` would be the common choice.